docs/src/man/querying_frameworks.md

```diff
@@ -247,3 +247,81 @@ These examples only scratch the surface of what one can do with
 referred to the [Query.jl
 documentation](http://www.queryverse.org/Query.jl/stable/) for more
 information.
+
+## DataFrameMacros.jl
```

Comment:
I would put this section before Query.jl not to break the flow of thought (as in the text you assume that the reader roughly remembers what DataFramesMeta.jl does).

Reply:
ok I'll move it